dpe/src/commands/sign.rs

```diff
@@ -282,6 +333,45 @@ mod tests {
         assert!(sig.verify(&TEST_DIGEST, &ec_pub_key).unwrap());
     }
 
+    #[test]
+    fn test_symmetric() {
+        let mut dpe = DpeInstance::<OpensslCrypto>::new_for_test(
+            Support {
+                auto_init: true,
+                is_symmetric: true,
+                ..Support::default()
+            },
+            &TEST_LOCALITIES,
+        )
+        .unwrap();
+
+        for i in 0..3 {
+            DeriveChildCmd {
+                handle: ContextHandle::default(),
+                data: [i; DPE_PROFILE.get_hash_size()],
+                flags: DeriveChildCmd::MAKE_DEFAULT,
+                tci_type: i as u32,
+                target_locality: 0,
+            }
+            .execute(&mut dpe, TEST_LOCALITIES[0])
+            .unwrap();
+        }
```

Comment:
I might be under thinking this, but I don't think we need to derive any children because it has the auto-initialized context